myteacher.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My teacher's technology</title>     
+    </head>
+    <body>
+        <h1>My teacher's technology stack</h1>
+        <ol>
+            <li>Ruby</li>
+            <li>Ruby on rails</li>
+            <li>Linux</li>
+            <li>JavaScript-Nodejs</li>
+            <li>Reactjs</li>
+        </ol>
+
+    </body>
```

Comment:
you have not closed the html tag
```
<html>
</html>
```

so add the closing tag, then push the work again

Reply:
also rename the file to be more meaningful it is talking about technology stack

so maybe the name should be `teacher-technology-stack.html`

`git mv myteacher.html teacher-technology-stack.html`

Reply:
ok let me do